source/Fleet.cpp

```diff
@@ -319,6 +319,8 @@ void Fleet::Enter(const System &system, list<shared_ptr<Ship>> &ships, const Pla
 			bool isWelcomeHere = !system.GetGovernment()->IsEnemy(government);
 			for(const System *neighbor : (hasJump ? system.JumpNeighbors(jumpDistance) : system.Links()))
 			{
+				if(!government->AllowJumpingTo(*neighbor))
```

Comment:
Should `unconstrained` fleets also be allowed to jump to restricted systems?

Reply:
You're right I forgot to add it for this function as well, I'll add it everywhere.